execute/plugin_functions.go

```diff
@@ -227,9 +227,9 @@ func mergeMessageObservations(
 	aos []decodedAttributedObservation, fChain map[cciptypes.ChainSelector]int,
 ) (exectypes.MessageObservations, error) {
 	// Create a validator for each chain
-	validators := make(map[cciptypes.ChainSelector]validation.MinObservationFilter[cciptypes.Message])
```

Comment:
If this is moved to `filter`, can it be axed from the `validation` package in exec?

Reply:
Thanks, I missed that. Yes, that was the intention.

Reply:
Ah you're right, I don't know why I expected the files to show up as deleted, they'd be just moved.